### DISCUSSION TITLE
ValueError: Pipeline does not contain nodes named ['preprocess_companies_node'].

### DISCUSSION THREAD

**Discussion by REIZER** [kedro-org/kedro#1933] [Idea]
I got an error when I ran it with the following URL.
https://kedro.readthedocs.io/en/stable/tutorial/create_pipelines.html

> kedro run --node=preprocess_companies_node
Where is the tutorial about node mentioned?

**Comment by stichbury** [comment]
I think the issue here is that your node is now in the `data_processing` namespace. So you need to issue the command

```
kedro run --node=data_processing.preprocess_companies_node
```

The reason you have the namespace is either because you've worked through the tutorial and introduced it ([in this step](https://kedro.readthedocs.io/en/stable/tutorial/namespace_pipelines.html)) or you built the Kedro starter for spaceflights and started with a full example, so the namespace was already there.

It's a problem with the docs that I'm in the process of solving. Hopefully my solution will help you!
